### PR TITLE
 New_samples (TTbarLepton_14 and QQH1252T_14) added without updating 'upgradeWorkflowComponents' file

### PR DESCRIPTION
**Pull request by sharmaaash**
#### PR description:

Samples "TTbarLepton_14 and QQH1252T_14" added. 
No additional changes in 'upgradeWorkflowComponents.py' file

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by chayanit** [comment]
Hello @cms-sw/generators-l2 , could we have this PR merged before the final 11_3_0 release? together with  #33414 , #33433 and #33386

File: Configuration/Generator/python/TTbarLepton_14TeV_TuneCP5_cfi.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                         maxEventsToPrint = cms.untracked.int32(0),
+                         pythiaPylistVerbosity = cms.untracked.int32(0),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         comEnergy = cms.double(14000.0),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        processParameters = cms.vstring(
+            'Top:gg2ttbar = on ',
+            'Top:gg2ttbar = on ',
```
Comment (agrohsje) [inline Configuration/Generator/python/TTbarLepton_14TeV_TuneCP5_cfi.py:16]: what is the reason to have this line twice? is this supposed to be qqbar ?

File: Configuration/Generator/python/QQH1252T_14TeV_TuneCP5_cfi.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         # put here the efficiency of your filter (1. if no filter)
+                         filterEfficiency = cms.untracked.double(1.0),
+                         comEnergy = cms.double(14000.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         # put here the cross section of your process (in pb)
+                         crossSection = cms.untracked.double(0.388),
+                         maxEventsToPrint = cms.untracked.int32(3),
```
Comment (agrohsje) [inline Configuration/Generator/python/QQH1252T_14TeV_TuneCP5_cfi.py:12]: Suggest to set to 0 for relval.

**Comment by agrohsje** [comment]
Please see my comments on the fragments. About names. Why is it called TTbarLepton ? Instead of QQH1252T I would propose QQToHToTauTau_mh125 or so.

File: Configuration/Generator/python/QQH1252T_14TeV_TuneCP5_cfi.py (hunk 2 above)
Comment (sharmaaash) [reply Configuration/Generator/python/QQH1252T_14TeV_TuneCP5_cfi.py:12]: Do you mean cross section or which parameter to set to zero?

**Comment by sharmaaash** [comment]
Hi,
I am sorry for the delaying due to urgent medical emergency of Covid. I
will follow all relevant suggestions and implement it by tomorrow.

**Comment by agrohsje** [comment]
@sharmaaash now the top is called higgstotautau.  I think you were misreading my comment. :-) 
What about: 
 QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py
 TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py
 Fragments are ok.

**Comment by sharmaaash** [comment]
> QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py

I made changes.

**Comment by agrohsje** [comment]
+generators

**Comment by qliphy** [comment]
+1

File: Configuration/Generator/python/QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
```
Comment (chayanit) [inline Configuration/Generator/python/QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py:3]: Hello, we noticed that this import has a wrong path and failed in production. It should be fixed soon. 
`from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *`

File: Configuration/Generator/python/TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
```
Comment (chayanit) [inline Configuration/Generator/python/TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py:3]: same here
we noticed that this import has a wrong path and failed in production. It should be fixed soon. 
`from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *`

Comment (qliphy) [reply Configuration/Generator/python/TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py:3]: > So do I need to create another PR with the following changes with the replacement ---> from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
> ?

Yes, please.

Comment (sharmaaash) [reply Configuration/Generator/python/TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py:3]: I made following changes and recreate PR:
**https://github.com/cms-sw/cmssw/pull/33700**

File: Configuration/Generator/python/QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py (hunk 3 above)
Comment (silviodonato) [reply Configuration/Generator/python/QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py:3]: see  https://github.com/cms-sw/cmssw/pull/33700

**Pull request merged**